src/pages/Home.js

```diff
@@ -0,0 +1,52 @@
+import React, { useEffect } from 'react';
+import { useDispatch, useSelector } from 'react-redux';
+import { Link } from 'react-router-dom';
+import { getAllData } from '../redux/continent';
+import {
+  GridContainer, MapBackground, Arrow, Title, Card,
+} from '../components/Styles/ContinentsStyle';
+import Imagen from '../components/images/mapa-mundi.png';
+
+function Home() {
+  const continents = useSelector((state) => state.reduContinent);
+
+  const dispatch = useDispatch();
+  useEffect(() => {
+    dispatch(getAllData);
+  }, []);
+
+  return (
+    <div>
+      <MapBackground>
+        <Title>
+          Covid-19
+          <br />
+          Metrics
+        </Title>
+        <img src={Imagen} alt="mapa-mundi" width={400} />
+      </MapBackground>
+      <GridContainer>
+        {continents.map((continent) => (
+          <Card key={continent.id}>
+            <li style={{ position: 'relative', left: '4rem' }}>
+              <Link to={`/country/${continent.id}`}>
+                <Arrow />
+              </Link>
+            </li>
+            <li><h2>{continent.continent}</h2></li>
+            <li>
+              <strong>Cases:</strong>
+              {continent.cases}
+            </li>
+            <li>
+              <strong>Deaths:</strong>
+              {continent.deaths}
+            </li>
+          </Card>
+        ))}
+      </GridContainer>
+    </div>
+  );
+}
```

Comment:
Your Home page should have a list of categories that could be filtered using given parameters, e.g. category name. Please try and add this feature so that your project meets the requirements. You could filter by cases or deaths and rearrange your continent results depending on the filter 📝

Reply:
Hi @mateo951! Thanks for your detailed feedback! I have some doubts about this particular issue. If you check the country page you will see the filter that displays the country info according to the user election. My first idea was display the data in a third page but I haven't the time to apply it because I face some bugs that distract my attention from this. Isn't the requirement accomplished? I mean, I applied the filter in the country search. 🤔

Reply:
The requirement specifies that the filter needs to be on the home page but it's true that met the technical requirements 😁Kindly make sure to apply these types of technical requirements in the propper directory 📝 Please ask for another review and you should be good 🤓

Reply:
@mateo951 Thanks! I implemented on my Home page a search bar, I think it will cover the requirements. I will ask for another review but I will also appreciate your comments about it. ☺ 
![image](https://user-images.githubusercontent.com/92180054/175777425-21f5cdd7-3963-4d09-8626-2256a9b2972d.png)

Here is how it works...
![image](https://user-images.githubusercontent.com/92180054/175777501-3dad0963-5970-41ae-bc34-b607587447d5.png)

Reply:
Looks amazing 😎 Well done!